README.md

```diff
@@ -296,44 +236,28 @@ fondant <subcommand> --help
 
 <p align="right">(<a href="#top">back to top</a>)</p>
 
-## 🚧 Current state and roadmap
-
-Fondant is currently in the alpha stage, offering a minimal viable interface. While you should
-expect to run into rough edges, the foundations are ready and Fondant should already be able to
-speed up your data preparation work.
-
-**The following topics are on our roadmap**
+## 👭 Contributing
 
-- Non-linear pipeline DAGs
-- LLM-focused example pipelines and reusable components
-- Data lineage and experiment tracking
-- Distributed execution, both on and off cluster
-- Support other dataframe libraries such as HF Datasets, Polars, Spark
-- Move reusable components into a decentralized component registry
-- Create datasets of copy-right free data for fine-tuning
-- Create reusable components for bias detection and mitigation
+We welcome contributions of different kinds:
 
-The roadmap and priority are defined based on community feedback. To provide input, you can join
-[our discord](https://discord.gg/HnTdWhydGp) or submit an idea in our
-[Github Discussions](https://github.com/ml6team/fondant/discussions/categories/ideas).
+|                                  |                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|----------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
I think there is an extra hidden row that can be removed 

![Uploading image.png…]()

Reply:
This is the empty header. If we remove it, it will no longer render as a table.